types/utils.d.ts

```
@@ -16,7 +16,7 @@ export declare function customize(handler: HandlerType): ({ detail, sendResponse
  * @param {function} func
  * @return curried function
  */
-export declare function curry(func: Function): (...args: any[]) => Function | void;
+export declare function curry(func: Function): () => any;
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)